### PR TITLE
update package info

### PR DESCRIPTION
**Pull request by rcannood**
* mention czi funding in readme
* simplify readme
* rename getting started vignette
* update authors to reflect current status

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -39,15 +39,16 @@ Authors@R: c(
...
         email = "brian_schilder@alumni.brown.edu", 
         comment = c(ORCID = "0000-0001-5949-2191", github = "bschilder")
     ),
     person(
         "Chananchida", "Sang-aram",
         role = "ctb",
         comment = c(ORCID = "0000-0002-0922-0822", github = "csangara")
-    )
+    ),
+    person("Data Intuitive", , , "info@data-intuitive.com", role = "aut")
```
Comment (lazappi) [inline DESCRIPTION:51]: Do you need all the empty commas? Maybe sponsor is a better role than author? And/or for CZI?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,19 @@
...
-status](https://www.r-pkg.org/badges/version/anndataR.png)](https://CRAN.R-project.org/package=anndataR)
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
+[![CRAN status](https://www.r-pkg.org/badges/version/anndataR.png)](https://CRAN.R-project.org/package=anndataR)
+[![R-CMD-check](https://github.com/scverse/anndataR/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/scverse/anndataR/actions/workflows/R-CMD-check.yaml)
 <!-- badges: end -->
 
-`{anndataR}` aims to make the AnnData format a first-class citizen in
+**{anndataR}** aims to make the AnnData format a first-class citizen in
 the R ecosystem, and to make it easy to work with AnnData files in R,
 either directly or by converting it to a SingleCellExperiment or Seurat
```
Comment (lazappi) [inline README.md:11]: Not in the PR but while you are changing things:

```suggestion
either directly or by converting them to a SingleCellExperiment or Seurat
```

File: README.md [hunk 3]
```diff
@@ -27,142 +22,45 @@ Feature list:
 >
 > This package is still in the experimental stage, and may not work as
 > expected. You can find the status of development of anndataR on the
-> [feature tracking
-> page](https://anndatar.data-intuitive.com/articles/design.html#feature-tracking)
-> of the website.Please report any issues you encounter.
+> [feature tracking page](https://anndatar.data-intuitive.com/articles/design.html#feature-tracking)
+> of the website. Please report any issues you encounter.
```
Comment (lazappi) [inline README.md:26]: There could be a link to the issues page here if you want.

File: README.md [hunk 4]
```diff
@@ -27,142 +22,45 @@ Feature list:
...
-``` r
-adata <- read_h5ad(h5ad_path, to = "HDF5AnnData")
-```
+The best way to get started with **{anndataR}** is to explore the package vignettes (available at https://anndatar.data-intuitive.com/articles/).
 
-View structure:
+- **Getting started**: An introduction to the package and its features.  
+  `vignette("anndataR", package = "anndataR")`
+- **Design**: An overview of the design of the package.  
+  `vignette("design", package = "anndataR")`
```
Comment (lazappi) [inline README.md:60]: Add links to the pkgdown site (if it's working)

File: README.md [hunk 5]
```diff
@@ -27,142 +22,45 @@ Feature list:
...
-#>     var: 'String', 'n_cells_by_counts', 'mean_counts', 'log1p_mean_counts', 'pct_dropout_by_counts', 'total_counts', 'log1p_total_counts', 'highly_variable', 'means', 'dispersions', 'dispersions_norm'
-#>     obsm: 'X_pca', 'X_umap'
-#>     varm: 'PCs'
-#>     layers: 'counts', 'csc_counts', 'dense_X', 'dense_counts'
-#>     obsp: 'connectivities', 'distances'
-```
+## Funding
 
-Access AnnData slots:
+This project is supported by the Chan Zuckerberg Initiative DAF, an advised fund of Silicon Valley Community Foundation.
```
Comment (lazappi) [inline README.md:64]: Maybe we should check with scverse if there is anything else that should go here? Or just add something like:

```
The scverse organisation receives [additional support](https://scverse.org/about/).

File: _pkgdown.yml [hunk 6]
```diff
@@ -1,4 +1,5 @@
-url: https://scverse.org/anndataR/
+url: https://anndataR.data-intuitive.com
```
Comment (lazappi) [inline _pkgdown.yml:1]: I guess we never really sorted out where this should go...

File: vignettes/anndataR.Rmd [hunk 7]
```diff
@@ -0,0 +1,155 @@
+---
+title: "Using anndataR to read and convert"
```
Comment (lazappi) [inline vignettes/anndataR.Rmd:2]: I would change this to either "Using anndataR to read and convert H5AD files" or just "Using anndataR"

File: vignettes/anndataR.Rmd [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+``` r
+library(anndataR)
+
+h5ad_path <- system.file("extdata", "example.h5ad", package = "anndataR")
+```
+
+Read an h5ad file in memory:
+
+``` r
+adata <- read_h5ad(h5ad_path)
```
Comment (lazappi) [inline vignettes/anndataR.Rmd:57]: I thought we made this function return a `SingleCellExperiment`/`Seurat` directly?

**Review by lazappi — APPROVED** [review]
I had some questions but they are they aren't really about this PR. I think this should be fine.

File: _pkgdown.yml (hunk 6 above)
Comment (rcannood) [reply _pkgdown.yml:1]: I think it was -- originally scverse.org was okay ( https://github.com/scverse/anndataR/issues/66 ), but then the core developers changed their mind and wanted us to host it elsewhere instead. :)

File: vignettes/anndataR.Rmd (hunk 8 above)
Comment (rcannood) [reply vignettes/anndataR.Rmd:57]: you can with `to = c("InMemoryAnnData", "HDF5AnnData", "SingleCellExperiment", "Seurat")`. SCE/Seurat was never the default, though.

**Pull request merged**